### DISCUSSION TITLE
Who's Hiring (May 2023)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#49471] [Show and tell]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #49472.

*If you're looking for last month's discussion, see #47869. Several companies [hired someone](https://twitter.com/nutlope/status/1610667757565083648) as a result of it!*

**Comment by brentarcane** [comment]
Cocoon Data - https://cocoondata.com | Frontend Engineer (Website) | Full time - Competitive salary | 80% Work from home - Sydney, Australia

**About us**

At Cocoon Data, we are pioneers in the ever-evolving world of digital security, committed to safeguarding the privacy and data of our clients in the face of rapidly advancing threats. We are proud to be building an innovative encryption platform that harnesses the power of quantum-resistant cryptography, ensuring robust protection for your most valuable digital assets.

Our mission is to weave a secure digital cocoon around our clients, shielding them from the unrelenting challenges of today's digital landscape. Our talented team of experts is dedicated to staying one step ahead of emerging cybersecurity threats by continuously pushing the boundaries of technology.

Join us at Cocoon Data, and be part of a team that is shaping the future of cybersecurity, empowering organisations to thrive in an increasingly complex digital world, safe in the knowledge that their data is secure and protected!

**This role**

We're completely overhauling our website from Wordpress to a custom Next.js application styled with TailwindCSS. We need someone to take ownership of the new Next.js application to help build and launch our new website and blog. You'll have a tonne of autonomy and creative freedom in creating your own designs and content, but have a small and focussed team to assist as and when required.

**Please note we are currently looking for Sydney based candidates at this time.**

**Individuals only please. Please do not contact me if you represent an agency.**

Please send an email to info@cocoondata.com with your details

**Reply by sumitshinde-84** [reply]
hii , 
My name is Sumit Shinde, and I am a full-stack developer from India. I am currently looking for a job opportunity and came across your company. As you mentioned, I have sent you an email expressing my interest in your project .

**Reply by brentarcane** [reply]
> Will you full remote anywhere if have Australian client reference?

Sorry, this is a hybrid role and we're looking for someone Sydney based right now.

**Reply by Ckbhatia** [reply]
Do you sponsor a VISA?

**Reply by nicksan222** [reply]
Hi, do you sponsor for a VISA? I am willing to relocate in Australia

**Comment by legions-developer** [comment]
Hello, i am a frontend dev from India! and I can do frontend work for your website indeed I'm creative and you can see my project works so checkout my skills 

I have sent you email, my email is gurbindersingh8727@gmail.com

**Reply by moinulmoin** [reply]
Brother, this section is for 'Hiring'. Please remove it and go to 'Want to be hired' section.

**Comment by stevo2105** [comment]
[Whop](https://whop.com/) | Senior Full-Stack Engineer | In person (New York, NY) | Full-time | $170-220k base | [Link](https://jobs.polymer.co/whop/26288)

Whop is a marketplace where people sell access to communities and software they create. We process over $10M / month and have almost a million users. Whop is hiring across the board. We are venture backed and looking for incredibly talented folks to join us in our NYC office. We are located in Brooklyn.

We have a small engineering team where you will be able to make incredible impact. Please reach out to me on [Twitter](https://twitter.com/cultured) or apply through the link above.

We are only hiring for in person roles at this time so please don't reach out unless you are willing to move to NYC. We have a massive opportunity on our hands and need to be completely aligned!

**Comment by minionhub** [comment]
[For Hire] [Full Time / Part Time] [Full Remote]
I am a highly skilled and experienced Freelance Full-Stack JavaScript Developer currently seeking new opportunities!

🔧 Skills & Expertise:
Languages : HTML5, CSS3, JavaScript, ECMA6, Typescript, PHP, Python.
Libraries : Gatsby, React.js, Next, Vue.js, Vuetify, Svelte, qwik, Nuxt
Frameworks : Wordpress, Angular, Laravel, Ruby on Rails, Django.
Frontend UI Templates : Bootstrap, Tailwind CSS, Material-UI, Semantic-UI, Ant Design.
Backend: Node.js, Nest.js, Prisma, GraphQL.
Database : MySQL, MongoDB, PostgreSQL, SQLite, Firebase.
Testing: Jest, Mocha, Jasmine, Cypress.
Issue Tracking Tools : Jira, Trello.
Version Control: Git, GitHub, GitLab, Bitbucket.
Payment Integration: Stripe.
Deployment: Digitalocean, Vercel.
Server Hosting : AWS(EC2, Lambda, S3, RDS, DynamoDB, ...), GCP, DigitalOcean.

🌐 Experience:
With over 10 years of experience in the industry, I have had the opportunity to work on a wide range or projects, from small startups to large-scale enterprise applications. My diverse skillset allows me to adapt to various project requirements and deliver high-quality results on time.

🤝 What I Offer:
Collaboration with your team to define requirements, timelines, and milestones
High-quality, clean, and efficient code that adheres to best practices
Regular communication and progress updates
Quick response time to questions and issues
Flexibility in working hours and timezone requirements

💼 Looking for:
Short-term or long-term projects
Full-time or part-time engagements
Remote work

✉️ Contact:
If you're interested in discussing a potential project, feel free to send me a DM or reach out via email at [robinson.james0427@gmail.com](mailto:robinson.james0427@gmail.com)!

I'm excited to hear about your projects and contribute to your success! Let's build amazing things together!

**Comment by minionhub** [comment]
Yes.

**Comment by chickencoder** [comment]
**VEED.IO** | Senior Full Stack Engineer | 100% Fully Remote (EU timezone) | Full-time | Competitive Salary | [Link](https://veed.teamtailor.com/en-GB/jobs/2605333-senior-full-stack-engineer-acquisition)

Check us out https://veed.io

---

At **VEED**, our mission is to make video creation accessible to anyone.

No hard-to-use equipment, painful learning curve, or keeping track of multiple subscriptions to take your ideas from record → share. With VEED, you can do this with a single tool.

In just a few years, we’ve gone from one user to millions. From no revenue to $12M+ ARR. From bootstrapped to [backed by Sequoia](https://techcrunch.com/2022/02/02/veed-an-online-video-editing-platform-picks-up-35m-from-sequoia/), one of the biggest VC firms in the world. From a team of two to a team of 160+. And we’re seeking more talented, motivated and entrepreneurial individuals to join our growing team.

---

Over the past 4 years, VEED has grown its web traffic from zero to millions of monthly visitors. This can be largely attributed to our world-class SEO engine, and we are constantly looking for ways to expand our reach and bring more users to our platform.

The Acquisition team's main focus is to create and iterate on products that bring new users to our platform. This includes developing web apps like our Video Compressor, Video Converter and working on our high impact SEO landing pages.

The team has a “startup within a startup” kind of culture. We value being able to run experiments and ship new initiatives quickly.

Feel free to reach out to me on [Twitter](https://twitter.com/JessetheSibley) if you have any questions 😊

**Reply by koleajeolayinka** [reply]
Would it be possible for someone from **Nigeria** to work remotely for your company

**Reply by thomaspapp28** [reply]
I am from Singapore. Can I Join?

**Reply by CharlieSay** [reply]
Can vouch for VEED. Great product.

**Comment by salahawk** [comment]
Only experienced developers only!!!! 
- hourly rate no higher than $10)
- 100% remote
- immediate start

frontend :next.js + taildwind css
backend : node.js (nest.js) + postgreSQL
hosting : digital ocean + cloudflare + nginx

**Reply by swarajbachu** [reply]
swarajbachu.tech

please can you tell, how I can react out to you

**Reply by sumitshinde-84** [reply]
Hii, I am interested .
Please contact me through 
shindesumit217@gmail.com,
https://twitter.com/Code_with_Roni

**Reply by thomaspapp28** [reply]
Hi, I am interested in how can I reach you

**Reply by enyelsequeira** [reply]
$10 per hour? No one should take this job, that’s a joke. Doesn’t matter where you are located

**Reply by Ckbhatia** [reply]
Suitable for interns in low pay countries

**Reply by sumitshinde-84** [reply]
> $10 per hour? No one should take this job, that’s a joke. Doesn’t matter where you are located

For me, the value of money pales in comparison to the opportunity to immerse myself deeply in the industry. This is an extraordinary chance for me to grow and excel

**Reply by Ayushsanjdev** [reply]
Big joke!

**Reply by richi-coder** [reply]
richicoder@gmail.com

**Comment by gabriella-anjani** [comment]
ApyHub - [apyhub.com](https://apyhub.com/) | Technical Product Manager | Remote  | Full-time |  [Link](https://apyhub.bamboohr.com/careers/37)

Are you on a mission to design and develop brilliant software and applications? Well then, you’re in the right place!



Why ApyHub? 

We believe in empowerment. In fact, in ApyHub everybody is given the keys to drive the company forward.

ApyHub strives to ensure that everyone is encouraged to identify and propose ideas and solutions. In principle, everyone at ApyHub knows exactly what they are responsible for and has the freedom to meet those expectations however they think is best!

We are committed to working hard to see how we can evolve and scale the way we manage self-organization and to finding ways to develop our culture and  core values in a way that works for everyone in the company. 



About the role

At ApyHub, we have a simple purpose: to be the place that software developers go to when they look for efficiency & simplicity. As a Technical Product Manager, your mission is to unlock engineering productivity at ApyHub.

You will work directly with your global engineering team and  our CTO to drive new technology innovation, design requirements, and continuous improvement of the platform. In more detail, you will:

Translate high-level business goals into an actionable, data-driven product roadmap
Actively participate in roadmap planning and help align product initiatives with business objectives and strategies
Create product requirements that bring clarity, focus and drive solutions to complex problems
Conduct market research and analyze the competitive landscape to ensure our platform meets our user needs
Work closely with marketing and developer relations to ensure that the product roadmap is aligned with the feedback from the community


You will be a strong candidate if you…

Have proven success in product management, with a track record of launching products and features that have met customer needs and business goals
Have minimum 3 years of experience building products for engineering teams (either as a product manager, a software developer or an architect)
Have excellent communication and presentation skills
Are very big on data-driven decisions, enthusiastic, curious, and a big fan of technology
Have knowledge of software engineering and engineering practices
Have a firm understanding of the entire software development lifecycle (SDLC)
Have analytical and problem-solving skills
Have experience working in an Agile environment 


As our new Technical Product Manager you’ll get

🖥  The chance to work on some of the most exciting technologies and build platforms that revolutionize the way people work
💃  The opportunity to work with a super fun, cool, talented team in an international environment
🎁  A competitive compensation package including other benefits such as equity 
📈  Learning, growth and career development budget
🤩  The moment to bring your ideas to live and experiment
🕰️  Flexible working hours and work location.. We care about getting stuff done rather than formalities
🗺  🇳🇱 The possibility of working remotely or in the Netherlands 

Got your attention?

Everything starts with getting in touch. The best way to express your interest is by applying online. We will be carefully reading your profile and can’t wait to hear from you!
Please apply through our [career page](https://apyhub.bamboohr.com/careers/37) 

What’s next?

After our team receives your application, and if you’re selected, you will have:

A 30 min video-call with our team to better understand your career goals and fit with the team
A technical challenge to help us evaluate your technical  skills, depending on the role you are applying for
A final video-call with the co-founders
We are doing our best to process your application in 10 working days and offer to follow within 24 hours. We can’t wait to meet you! 

If you have any questions, please reach out to me on my [LinkedIn](https://www.linkedin.com/in/gabriella-valonia-anjani/), thank you!

**Comment by fofiu** [comment]
[Monogram](http://monogram.io/) | Senior Next.js Engineer | Remote (North & South America) | Full-time | https://monogram.io/careers

**Introduction**

You don’t work at Monogram to remove roadblocks, you work here to pave new paths. You don’t just solve problems, you join a family of designers & developers who collaborate like clockwork, rise together, fall together, and do their life’s work here.

As part our strong, fast moving team that’s been a [Vercel partner](https://vercel.com/experts) from early days, you will be helping convert beautifully designed web applications into reality, including building performant APIs and modern front-ends using Next.js.

Your role also includes coordinating with the teams responsible for other layers of the application design & infrastructure.

**Duties**

- Collaborate on a daily basis with a team of product managers, engineers, and UI/UX designers to understand business requirements, user experience goals and pain points
- Ability to develop websites/web apps and backend APIs
- Have a strong understanding of Core Web Vitals and how code impacts performance
- Build reusable components and libraries for future use

**🚨 Required Skills**

- TypeScript
- Next.js / React / Preact
- Solid Node.js experience
- GraphQL & REST API concepts and implementation patterns
- Testing frameworks such as Jest, Playwright, etc.
- Auth0 and other authentication services
- Content modeling in CMS’es such as Contentful, Dato, Prismic, Sanity, etc.
- Quality/clean code
- 2+ years of experience working in a fast-paced development environment using React and any of the following (Svelte, Angular, Vue, etc.)

**💭 What’s not required**

- HTML/CSS perfection — usually HTML/CSS screens will be provided by a front-end developer focused solely on perfecting the UI in HTML/CSS

**🧁 Bonus**

- Svelte, SvelteKit
- Understanding of middleware concepts in Next.js
- A/B testing implementation
- PostgteSQL/RDBMS systems
- Understanding of API Security
- Stripe

**💎 Even better**

- Python and some data science concepts
- Three.js

If interested, please send us an email with your resume (and/or a video introduction) to [careers@monogram.io](mailto:careers@monogram.io)

**Reply by sumitshinde-84** [reply]
Interested

**Reply by Apestein** [reply]
👀

**Reply by sachitrana2001** [reply]
I'm from Indian. Interested....

**Comment by billpull** [comment]
[ParlayPlay](https://parlayplay.io/) | Senior FullStack Engineer | Remote (North America Preferred) | Full-time | [careers@parlayplay.io](mailto:careers@parlayplay.io)

**Who We Are**
Launched in 2021, ParlayPlay is a social-first daily fantasy sports operator. As our first technical hire, you have the opportunity to join at the foundation of a fast growing startup, in a booming industry! There has never been a better time to join the fantasy sports and gaming industry. This is a critical role, providing an opportunity to work directly with the founding team, with an ability to have real impact from day one. This role reports directly to the VP Engineering (Co-Founder). 

**The Role**
As a Senior Full Stack Engineer at ParlayPlay, you will have the opportunity to work on exciting projects and contribute to the development of the company's future vision. You will be part of a collaborative team of engineers who are passionate about delivering innovative and scalable products. Your role will be critical in ensuring that ParlayPlay's products continue to meet the evolving needs of our customers. Your expertise in software development, combined with your ability to work effectively with cross-functional teams, will enable you to bring new and innovative ideas to life. In this role, you will have the chance to take ownership of projects, from conception to deployment, and make a real impact. ParlayPlay offers a dynamic and fast-paced environment where you can challenge yourself, grow your skills, and make meaningful contributions.

**Technologies**

- Python
- Django
- Postgresql 
- React (Next JS)
- Typescript
- AWS
- Docker

**What You’ll Be Working On**
Deliver high quality solutions and features with speed and end-to-end ownership of your projects.
Collaborate cross-functionally with team members to further the platform and vision.
Implement best practices for automated testing, coding guidelines, and CI/CD.
Provide ongoing maintenance, support and enhancements in existing systems and platforms.

**Skills / Experience**

- 4+ years Python experience, preferably in Django or similar web frameworks.
- 4+ years Experience working with React and Javascript, proficiency in Typescript preferred.
- Experience writing testable, highly available web applications/services.
- Proficient with relational databases such as PostgreSQL.
- Comfortable working on complex problems in a fast paced early stage environment.

**Additional Skills**

- Understanding of security concerns when building Web APIs.
- Understanding of containerized applications.
- Familiarity with Next.JS
- Familiarity with AWS, specifically ECS.
- Familiarity with Cloudflare.
- Familiarity with Terraform.

**Reply by crazydevlegend** [reply]
I'm happy to apply on this opportunity.
I'm currently running my start-up and working as a Lead Developer (this is part-time)
And working as a full-time freelancer. Kindly check my profile and ping me if interested. 🙏

**Comment by sbmadhav** [comment]
Hello! We are Hiring!

Join us at The [Ascension](https://ascension.org/) Studio as a Sr. Architect of Experience Technology specializing in frontend development for omnichannel experiences. We use Next.js for the Web and React Native for Mobile Apps. We use GraphQL with Apollo Client for accessing data. We work with a design system heavily inspired by material UI and use BrowserStack for cross-platform testing.

If you're looking for a new challenge and a chance to work closely with a supportive team, this role might be perfect for you. Let's connect and discuss the opportunity further!

This is your chance to impact patients' lives through innovative web and mobile applications.

This job is entirely remote in the US only.

Apply here: [https://ascensionjobs1-ascension.icims.com/jobs/239446/sr.-architect%2c-experience-technology-%28front-end%29/job?mode=view](https://ascensionjobs1-ascension.icims.com/jobs/239446/sr.-architect%2c-experience-technology-%28front-end%29/job?mode=view)

**Reply by thomaspapp28** [reply]
I am very interested in this opportunity.

**Reply by sbmadhav** [reply]
Please apply using the link in the description above.

**Reply by koleajeolayinka** [reply]
@sbmadhav On my end, when I access the link, it only shows a logo without any other content.

![image](https://github.com/vercel/next.js/assets/100846558/323b6276-4bff-4942-8b38-db48d57c70c4)

**Reply by sbmadhav** [reply]
It should work if you are in the US. I just tried in two different devices to confirm that I am able to open. That is the only and direct link to apply.

**Comment by creteurlouis** [comment]
**Acoustic Wells| Senior Full-Stack Engineer | Remote (Remote or Boston) | Full-time or Part-time | www.acoustic-wells.com/careers**

At Acoustic Wells we are a fast paced, dynamic team dedicated to experimentation and growth.
We manage key pieces of Oil & Gas infrastructure using IOT, Machine learning & Data Science.
We are rapidly expanding, and value key partners who can move with us as we grow and learn.
We’re looking for a forward-thinking Senior Front-end Engineer to help build out our different dashboards, APIs, Web applications and also work on the global architecture of our solutions.
Are you ready to hop on board an exciting, MIT born, Data Science & IoT startup? Are you ready to help drive innovation and not be a “ticket puncher”?
Step up and let’s do this!


**Who We Are**
We are a startup in the energy and IoT space, based on research done at MIT. We are developing a solution to help oil and gas producers reduce their environmental footprint and increase their field efficiency by allowing them to automatically control and monitor their assets remotely.

**Technologies**
Python
FastAPI
React (Next JS)
Typescript
.NET
AWS
Material UI
Postgresql

**Skills / Experience**
4+ years of experience working with React/Next.js using Typescript
3+ years of experience with .NET or Python (Flask / FastAPI or Django)
PostgreSQL
MUI

Apply Here:  www.acoustic-wells.com/careers

**Comment by thomaspapp28** [comment]
I am interested in.

**Reply by thomaspapp28** [reply]
The link is not available.

**Comment by SwoopAgency** [comment]
Hello Github community,

I'd like to introduce you to Swoop, your one-stop destination for all things branding, community management, graphic design, web development, marketing, and content creation.

At Swoop, we are passionate about helping businesses and individuals elevate their online presence. Whether you need a stunning website, eye-catching graphics, an engaging social media strategy, or captivating content, we've got you covered.

Our experienced team is here to bring your vision to life and help you stand out in the digital world. Let's soar together with Swoop!

For more information, feel free to contact us on [swoop.agencia@gmail.com](mailto:swoop.agencia@gmail.com). We're here to turn your digital dreams into reality. :)

Thank you for your time and trust in Swoop.

Best regards, Alex
